<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowserWindow</class>
 <widget class="QDockWidget" name="FileBrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Browser</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../../../res/OpenCOR.qrc">
     <normaloff>:/home.png</normaloff>:/home.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Home</string>
   </property>
   <property name="statusTip">
    <string>Go to the home folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Home</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../../../res/OpenCOR.qrc">
     <normaloff>:/back.png</normaloff>:/back.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="statusTip">
    <string>Go to the previous folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../../res/OpenCOR.qrc">
     <normaloff>:/forward.png</normaloff>:/forward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="statusTip">
    <string>Go to the next folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Right</string>
   </property>
  </action>
  <action name="actionParent">
   <property name="icon">
    <iconset resource="../../../res/OpenCOR.qrc">
     <normaloff>:/up.png</normaloff>:/up.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Parent</string>
   </property>
   <property name="statusTip">
    <string>Go to the parent folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Up</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../res/OpenCOR.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create a new folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="text">
    <string>Folder</string>
   </property>
   <property name="statusTip">
    <string>Create a new folder</string>
   </property>
  </action>
  <action name="actionNewCellML10File">
   <property name="text">
    <string>CellML 1.0 File</string>
   </property>
   <property name="statusTip">
    <string>Create a new CellML 1.0 file</string>
   </property>
  </action>
  <action name="actionNewCellML11File">
   <property name="text">
    <string>CellML 1.1 File</string>
   </property>
   <property name="statusTip">
    <string>Create a new CellML 1.1 file</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../res/OpenCOR.qrc">
     <normaloff>:/delete.png</normaloff>:/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="statusTip">
    <string>Delete the current folder or file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../res/OpenCOR.qrc"/>
 </resources>
 <connections/>
</ui>
